<launch>
   
    <node name="gripper_server" pkg="clean_robot" type="gripper_server.py" output="screen" />
    <node name="gripper_client" pkg="clean_robot" type="gripper_client.py" output="screen" />
</launch>
